index.bs

```diff
@@ -782,11 +782,15 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 
 		[SecureContext, Exposed=Window]
 		interface Clipboard : EventTarget {
-			Promise&lt;ClipboardItems&gt; read();
+			Promise&lt;ClipboardItems&gt; read(optional ClipboardUnsanitizedFormats formats = {});
```

Comment:
Why is this a dictionary with a single member which is an array?

If this is a dictionary simply because it makes it easier to extend with more parameters later, then I'd have expected the dictionary to be called `readOptions` or something, with an array called `unsanitizedFormats`.

Reply:
If we just have `sequence` with `DOMString`, then we cannot make it optional (at least the bindings generator fails in Chromium). If we make it non-optional, then it would break sites that do not want to read unsanitized formats. Dictionary helps us to make this arg optional and `unsanitized` makes it explicit that authors are opting into reading unsanitized formats i.e. when sites call `navigator.clipboard.read({unsanitized: ['text/html']})`, it is clear to the reader that the site is trying to read unsanitized `text/html` format. We got [this feedback ](https://github.com/w3c/editing/issues/315#issue-942457383)from Chrome's security team.